package.json

```diff
@@ -3,5 +3,9 @@
   "private": true,
   "devDependencies": {
     "lerna": "^3.20.2"
+  },
+  "scripts": {
+    "init": "yarn && lerna bootstrap --force-local && lerna link",
```

Comment:
[`lerna bootstrap`](https://github.com/lerna/lerna/tree/master/commands/bootstrap) should install dependencies and symlink all of the packages so `yarn` and `lerna link` shouldn't be necessary.

Reply:
Why `--force-local`? I haven't worked with lerna very much so I'm not sure if this is common but I would think our versions should match when running `init`.

> When passed, this flag causes the bootstrap command to always symlink local dependencies regardless of matching version range.

Reply:
first yarn is needed to install top level packages. `lerna` call will fail without it, except case when you have installed lerna -g.

`--force-local` is not mandatory but due to `^` version notation in packages we can catch an edge case when newer version of package being published to npm, but not merged in main brunch. or you in the middle of development of some feature and do not want or forget to rebase and need to run `init`. I am not sure in how realistic this scenarios. With `--force-local` you avoid that external package being fetched and\or cached somewhere, and you end up with making changes, and do not seeing them while hot-reload. 

additional lerna link is also not mandatory but, at least on my machine I encountered cases when I need it additionally after bootstrap. I don't know due to what it happening (npm caches, wrong moon phase?) but adding this as after bootstrap works pretty stable.

Reply:
Thanks for the explanation! Would you mind adding then instructions to get things started and the explanation of what the commands do to the README?